docs/how_to/chromosphere_plot.rst

```diff
@@ -0,0 +1,44 @@
+.. _sunpy-plot-chromosphere-model:
+
+***********************************
+Plotting the Chromosphere Model
+***********************************
+
+This example demonstrates how to load and visualize one of the models in `sunpy.sun.models`, in this case :cite:t:`avrett_loeser_2008` solar chromosphere model.
+Which we will just plot a range of parameters (temperature and density as a function of height).
+
+.. code-block:: python
```

Comment:
Does the figure show up in the rendered version of the documentation? 

If not, this might have to change to a plot block instead.